tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -637,8 +625,8 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
                 return nil, sharedSet, err
             }
         } else {
-            whisk.Debug(whisk.DbgError, "--kind argument '%s' is invalid\n", flags.action.kind)
-            errMsg := fmt.Sprintf("'%s' is not a supported action runtime", flags.action.kind)
+            whisk.Debug(whisk.DbgError, "Action runtime exention '%s' is not supported\n", ext)
```

Comment:
I believe there will be .zip file support in the future. That means a user would be able to specify a --kind swift, and upload a zip file.